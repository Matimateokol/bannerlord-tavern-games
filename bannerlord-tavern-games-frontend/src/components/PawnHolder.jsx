import Pawn from './Pawn';
import PropTypes from 'prop-types';

export default function PawnHolder({ id, pawnStyleName }) {
  return (
    <div className="pawnHolder">
      <Pawn pawnStyleName={pawnStyleName} id={id} />
    </div>
  );
}

PawnHolder.propTypes = {
  id: PropTypes.number,
  pawnStyleName: PropTypes.string
};
