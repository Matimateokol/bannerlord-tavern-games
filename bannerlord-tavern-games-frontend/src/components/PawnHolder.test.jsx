import { render } from '@testing-library/react';
import PawnHolder from './PawnHolder';

test('Test if PawnHolder matches snapshot with given prop values', async () => {
  const { pawnHolder } = render(<PawnHolder id={3} pawnStyleName={'blackPawn'} />);

  expect(pawnHolder).toMatchSnapshot(`
    <div className="pawnHolder">
      <button className='blackPawn'>3</button>
    </div>
  `);
});
