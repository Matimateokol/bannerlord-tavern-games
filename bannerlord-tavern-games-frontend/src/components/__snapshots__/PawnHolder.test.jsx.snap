// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if PawnHolder contains a Pawn with given props: 
  <div className="pawnHolder">
    <button className='blackPawn'>3</button>
  </div>
   1`] = `undefined`;

exports[`Test if PawnHolder matches snapshot with given prop values: 
    <div className="pawnHolder">
      <button className='blackPawn'>3</button>
    </div>
   1`] = `undefined`;

exports[`Test if PawnHolder matches snapshot with given prop values: 
  <div className="pawnHolder">
    <button className='blackPawn'>3</button>
  </div>
   1`] = `undefined`;

exports[`Test if PawnHolder matches snapshot with given props: 
  <div className="pawnHolder">
    <button className='blackPawn'>3</button>
  </div>
   1`] = `undefined`;
